app/views/category_views.mli

```diff
@@ -0,0 +1,16 @@
+module Category :
```

Comment:
I suggest to hide this module. 
(TBH, I'm not sure this submodule is useful here).